src/screens/SUE/SueReportScreen.tsx

```diff
@@ -210,11 +210,17 @@ export const SueReportScreen = ({
           const images = JSON.parse(getValues().images);
 
           let totalSize = 0;
-          images.map(async ({ size }: { size: number }) => {
+          const sizeOfImage = images.some(({ size }: { size: number }) => {
```

Comment:
this should be refactored to a name like `isImageGreater10MB` because the `some` returns a boolean @ardasnturk @d4v3d4vE!

Reply:
Ow, yeah, you're right. I will update this.

Reply:
Sure, I'll adjust it!